activesupport/lib/active_support.rb

```diff
@@ -110,6 +110,12 @@ def self.to_time_preserves_timezone
   end
 
   def self.to_time_preserves_timezone=(value)
+    unless value
+      ActiveSupport::Deprecation.warn(
+        "Support for the pre-Ruby 2.4 behavior of to_time has been deprecated and will be removed in Rails 7.1."
```

Comment:
Sorry, a bit late but shouldn't it be Rails 7.2/8.0 rather than 7.1?

Reply:
@robin850 you're absolutely right - happy for you to get an easy contributor point 😄